SapienNetwork/Sapien
# feat: Lerna Setup

What you should be able.

- yarn dev should start the local frontend

## Test Steps

- Go to `packages/frontend/pages/index.tsx` and simply do `import components from sapien-components'`; and you should see something like

<img width="348" alt="Screen Shot 2020-08-06 at 11 06 45 PM" src="https://user-images.githubusercontent.com/8289486/89608016-85569180-d839-11ea-89d2-abf009a1b62c.png">

This means lerna is running great with frontend & components
